fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -293,13 +340,29 @@ if [ ! -e "$fMRIFolder" ] ; then
   log_Msg "mkdir ${fMRIFolder}"
   mkdir "$fMRIFolder"
 fi
-cp "$fMRITimeSeries" "$fMRIFolder"/"$OrigTCSName".nii.gz
+${FSLDIR}/bin/imcp "$fMRIScout" "$fMRIFolder"/"$OrigScoutName"
+
+# --- Do slice time correction if indicated
+if [ $DoSliceTimeCorrection = "TRUE" ] ; then
+    log_Msg "Running slice timing correction"
+    SliceTimerCorrectionParameters=`echo ${SliceTimerCorrectionParameters} | sed 's/@/ /g'` 
+    ${FSLDIR}/bin/slicetimer -i "$fMRIFolder"/"$OrigTCSName" -o "$fMRIFolder"/"$OrigTCSName"_orig_stc -r ${TR} -v ${SliceTimerCorrectionParameters}
+    
+    if [ "${DistortionCorrection}" = "NONE"] ; then {
+      #  use FSL's fslreorient2std for reorienting the image to match the approximate orientation of the standard template images MNI152
+      #  this makes the single-band processing more robust to registration problems.
+      #  NOTE: Take into account that BOLD images might have a different spatial orientation!
+
+      log_Warn "Performing fslreorient2std! Please take that into account when using volume BOLD images in further analyses!"
+      ${FSLDIR}/bin/fslreorient2std "$fMRIFolder"/"$OrigTCSName"_orig_stc "$fMRIFolder"/"$OrigTCSName"
+    fi
```

Comment:
You don't have any code that copies the _orig_stc file to "$fMRIFolder"/"$OrigTCSName" if reorienting is skipped.

Reply:
added